src/containers/withAuthorizedAction/withAuthorizedAction.tsx

```diff
@@ -28,11 +42,44 @@ import {
 } from './withAuthorizedAction.types'
 
 const mapState = (state: RootStateOrAny): MapStateProps => ({
-  address: getAddress(state)
+  isMagicAutoSignEnabled: getIsFeatureEnabled(
+    state,
+    ApplicationName.DAPPS,
+    'magic-auto-sign'
```

Comment:
should we move this definition to a const/enum?

Reply:
Moved it.